### ISSUE TITLE
/add with output possible?

### ISSUE THREAD

**Issue by sasukebinbin** [alexta69/metube#373]
Thanks for creating such a great project. 

I can use https://metube.domain.com/add to submit the request. 

But would like to know is it possible to pass parameter like Output template to set the file name. 

Actually what I want is just Restricted file name which available in yt-dl. Eg. fixed the file name

Thanks

**Comment by PikuZheng** [comment]
yes... you can post `custom_name_prefix` and it will insert before filename. 

but I'm guessing you need to specify a filename directly? it needs a little sourcecode modification

**Comment by sasukebinbin** [comment]
> yes... you can post `custom_name_prefix` and it will insert before filename.
> 
> but I'm guessing you need to specify a filename directly? it needs a little sourcecode modification
.Yes Because I use google api to perform search songs then download as mp3. So I want to set the file as my search keywords.mp3.  Not adding prefix. And the length for the default title of the video is too long.

**Comment by PikuZheng** [comment]
but set OUTPUT_TEMPLATE to `.%(ext)s` at the same time？
